src/styles/main.scss

```diff
@@ -1,3 +1,6 @@
+$c-cardOnHover: #34568b;
+$c-linkOnHover: #00acdc;
```

Comment:
Create separate file for vars.

Reply:
I agreed with your suggestions and have done it all exactly this way
Thank you